model/github_runner.rb

```diff
@@ -13,14 +13,14 @@ class GithubRunner < Sequel::Model
   semaphore :destroy
 
   def run_url
-    "http://github.com/#{repository_name}/actions/runs/#{run_id}"
+    "http://github.com/#{repository_name}/actions/runs/#{workflow_job["run_id"]}"
   end
 
   def job_url
-    "http://github.com/#{repository_name}/actions/runs/#{run_id}/job/#{job_id}"
+    "http://github.com/#{repository_name}/actions/runs/#{workflow_job["run_id"]}/job/#{workflow_job["id"]}"
   end
 
   def runner_url
-    "http://github.com/#{repository_name}/settings/actions/runners/#{runner_id}"
+    "http://github.com/#{repository_name}/settings/actions/runners/#{runner_id}" if runner_id
```

Comment:
is this a related change?

Reply:
I think it's not. Let me move it to another PR

Reply:
moved